bread/layout/components/tile.py

```diff
@@ -85,20 +88,31 @@ def __init__(
 
         super().__init__(
             hg.BUTTON(
-                Icon("chevron--down", size="16"),
-                aria_label="expand menu",
-                _class="bx--tile__chevron",
-            ),
-            hg.DIV(
-                hg.SPAN(
-                    above,
-                    **above_attrs,
+                hg.DIV(
+                    Icon("chevron--down", size="16"),
+                    _class="bx--tile__chevron",
+                ),
+                hg.DIV(
+                    hg.SPAN(
+                        above,
+                        **above_attrs,
+                    ),
+                    hg.SPAN(
+                        below,
+                        **below_attrs,
+                    ),
+                    _class="bx--tile-content",
                 ),
-                hg.SPAN(
-                    below,
-                    **below_attrs,
+                onload=hg.format(
+                    (
+                        "(function(){{"
+                        "const thisTile = document.querySelector('#{}');"
+                        "window.setExpandableTileMaxHeight(thisTile);"
+                        "}})();"
```

Comment:
Use of querySelector and tile_id can be avoided if you use ```let that = this; (function()...``` and then use ```that``` inside the function.
Using id for components is always a bit problematic, as it is difficult to guarantee that the id is really unique (e.g. when a component is used inside an ```hg.Iterator```, the id will be the same for every element.

Reply:
Done.